tests/kernel/boot_ltp.pm

```diff
@@ -161,6 +162,13 @@ EOF
         script_run('ping6 -c 2 $IPV6_RNETWORK:$RHOST_IPV6_HOST');
     }
 
+    if (is_pvm) {
+        # Check and activate hugepages before test execution
+        assert_script_run 'grep -e Huge -e PageTables /proc/meminfo';
+        assert_script_run 'echo 1 > /proc/sys/vm/nr_hugepages';
+        assert_script_run 'grep -e Huge -e PageTables /proc/meminfo';
+    }
```

Comment:
Wouldn't be better to use `script_run`? And how about limit this setup just for the tests which needs that? (see `$is_ima`).

Reply:
Currently is just for SPVM and for whole LTP tests. I'd like it to be just for a test which needs that and on all archs. BTW shouldn't this go to upstream LTP test?

Reply:
OK, @metan-ucw suggest to keep it for all tests (there are more tests that use hugepages (at least `syscalls`, `hugetlb`, `mm`). This is a temporary solution, until it's fixed in LTP. + Use it for all archs.

Reply:
updated commit msg and `script_run`